### ISSUE TITLE
Aimbot just doesnt work

### ISSUE THREAD

**Issue by Zmanram12** [Jire/Overwatcheat#183]
I followed the video changed my config downloaded java changed the enemy team to magenta got interception dll did everything but I run the start bat it seems like its working but ingame it does nothing please help

**Comment by ipkstef** [comment]
What aim key are you using? Confirm the right key number with: https://github.com/Jire/VKey-Finder

You have to hold the key and the magenta needs to be within ~256x256pixels of the center of your screen. Practice in training

**Comment by i9xgit** [comment]
hey im having this issue too, i got the right vkey and magenta enemy colors but nothing

**Comment by FuYon** [comment]
I have the same issue too, all settings are right and it just doesn't work. Plus which version of jdk are you using, me is 18...

**Comment by FuYon** [comment]
> What aim key are you using? Confirm the right key number with: https://github.com/Jire/VKey-Finder
> 
> You have to hold the key and the magenta needs to be within ~256x256pixels of the center of your screen. Practice in training

Yeah, the right setting with the ami key 1, tested with soldier but sadly it doesn't work

**Comment by shlomivman** [comment]
> > What aim key are you using? Confirm the right key number with: https://github.com/Jire/VKey-Finder
> > You have to hold the key and the magenta needs to be within ~256x256pixels of the center of your screen. Practice in training
> 
> Yeah, the right setting with the ami key 1, tested with soldier but sadly it doesn't work
> 
> Maybe I could put my config here:
> 
> ```
> aim_key=1
> aim_mode=1
> sensitivity=15.0
> fps=144
> aim_duration_millis=8
> aim_duration_multiplier_base=5
> aim_duration_multiplier_max=2.0
> aim_max_move_pixels=5
> aim_jitter_percent=0
> aim_min_target_width=8
> aim_min_target_height=8
> box_width=256
> box_height=256
> max_snap_divisor=2.0
> target_colors=dd32db,db33d8,e23be9,d619dd,d200d6
> target_color_tolerance=16
> window_title_search=Fullscreen Projector
> device_id=11
> aim_offset_x=1.00
> aim_offset_y=0.75
> flick_shoot_pixels=5
> flick_pause_duration=350
> enable_overlay=false
> ```

your aim_duration_millis should be 7 instead of 8, don't know if that is the issue
